soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -20,7 +20,11 @@
 #endif
 
 /* Software reset defines */
+#ifdef CONFIG_SOC_RISCV_TELINK_B95
+#define reg_reset                   REG_ADDR8(0x14082f)
+#else
 #define reg_reset                   REG_ADDR8(0x1401ef)
+#endif
```

Comment:
Again use same style:
```
#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
	...
#elif CONFIG_SOC_RISCV_TELINK_B95
	...
#endif
```

Reply:
OK, will check all the define of the CONFIG_SOC_RISCV_TELINK_B95, to make sure all the code to follow the same style .